src/nbl/video/IGPUDescriptorSet.cpp

```diff
@@ -152,7 +169,10 @@ void IGPUDescriptorSet::dropDescriptors(const IGPUDescriptorSet::SDropDescriptor
 
 	// we only increment the version to detect UPDATE-AFTER-BIND and automagically invalidate descriptor sets
 	// so, only if we do the path that writes descriptors, do we want to increment version
-    if (getOriginDevice()->getEnabledFeatures().nullDescriptor)
+    auto& bindingRedirect = m_layout->getDescriptorRedirect(descriptorType);
```

Comment:
if you look at https://github.com/Devsh-Graphics-Programming/Nabla/blob/3df38a45978t510f9823c97adfbc3bea4c90e8aee/include/nbl/video/IGPUDescriptorSet.h#L89

it performs a `getDescriptorRedirect(type)` then calls https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/include/nbl/asset/IDescriptorSetLayout.h#L161

which means it calls `findBindingStorageIndex` alreayd

I'd like to amortize this lookup of a storage_offset` and use it to both `getCreateFlags` and get your destination arraydescriptor array